var/spack/repos/builtin/packages/py-wand/package.py

```diff
@@ -23,3 +24,6 @@ class PyWand(PythonPackage):
     depends_on("python@2.7:2.8,3.3:", type=("build", "run"))
 
     depends_on("py-sphinx@1:", type="build", when="+docs")
```

Comment:
Sphinx version required has changed. Honestly, we don't actually run any of the commands to build the docs, so maybe we should just remove this